src/services/ethService.js

```diff
@@ -256,7 +256,7 @@ async function getTxInfo(txHash) {
     value,
     asset,
     contractAddress,
-    status: txReceipt.status === '0x1' ? 'confirmed' : 'failed',
+    status: (txReceipt.status == '0x1') ? 'confirmed' : 'failed',
```

Comment:
why == and not ===?

Reply:
txReceipt.status is Bool.. so it fails type check